src/views/tables/OrdersTable.tsx

```diff
@@ -212,7 +212,6 @@ const getOrdersTableColumnDef = ({
         label: stringGetter({ key: STRING_KEYS.ACTION }),
         isActionable: true,
         allowsSorting: false,
-        getCellValue: () => '',
```

Comment:
I think the merge of main into this base branch was messed up and didn't get this change somehow.